speed-dreams/source-2.2.3/src/interfaces/track.h

```diff
@@ -367,6 +369,8 @@ typedef struct trackSeg
 #define TR_PITBUILDING	4       /**< Pit building wall (barrier only) */
 
     tdble length;               /**< Length in meters of the middle of the track */
+    /// SIMULATED DRIVING ASSISTANCE: added tdble time
+    tdble time;                 /**< Estimated time in seconds */
```

Comment:
You added this so make it follow our naming conventions,

Reply:
I changed this to Time cause we also use Uppercase for struct properties in other parts of our code. That good?